addons/escoria-core/game/core-scripts/esc/types/esc_command_argument_descriptor.gd

```diff
@@ -2,6 +2,15 @@
 extends Object
 class_name ESCCommandArgumentDescriptor
 
+# As the get_type command was deprecated with Godot 2.x w we need a way to determine
+# variable types. Ideally these wouldn't be hardcoded but there's no GDScript 3.x command to
+# turn a type back to its name.
+var godot_type_list = ["TYPE_NIL", "TYPE_BOOL", "TYPE_INT", "TYPE_REAL",  "TYPE_STRING", \
```

Comment:
Minor nitpick/comment/not married to it, but, unless this list can be added to dynamically, maybe this should be marked as `const`? I realize that arrays in GDScript are passed by reference anyway, but for the sake of intent (a la pseudo-private prefixes), maybe this makes some sense. Other than that, this PR looks good.

Reply:
If you can CONSTANT_CASE the name here, I think we're good!

Reply:
Made a couple of changes here in addition to CONSTANT_CASE. I've replaced the "|" with " or ", removed "TYPE_" off all the types,  and lower-cased all the types to make it more readable. New error example:
Argument 0 ("-300") is of type string . Expected [ int or real ]